README.md

```diff
@@ -29,7 +29,13 @@ import { RevAiApiClient } from 'revai-node-sdk';
 
 // Initialize your client with your Rev AI access token
 const accessToken = "Your Access Token";
-const client = new RevAiApiClient(accessToken);
+/**
+ * Optionally set the base url of the Rev AI deployment of your account, defaults to US.
+ * Note the base url for the streaming API is different from the asynchronouse API,
+ * use RevAiBaseWebsocketUrl and RevAiBaseUrl, respectively.
+ * Learn more about Rev AI's global deployments [here](https://docs.rev.ai/api/global-deployments/).
+ */
+const client = new RevAiApiClient({ token: accessToken, baseUrl: RevAiBaseUrl.US } );
```

Comment:
I don't think we should have `baseUrl` as the interface here. Customer's don't care about what url they are submitting to, but rather what region they are submitting to. Or if region is too specific, it could be something like deployment

Reply:
should we just make  BaseUrls available to customers and recommend using those constant? I personally actually like flexibility of specifying your own url if needed

Reply:
perhaps, but I think its a bit of an implementation leak. Take aws clients for example, you specify the region, you want, not the url. I think its more useful for streaming as well, so users don't have to swap out urls, but we can handle the switching internally

Reply:
I'm happy to discuss if you want

Reply:
I think base url gives us more flexibility if in the future we have more than 1 deployment per region, or if a customer has a deployment just for them

Reply:
I think in this case we could do `RevAiDeployment.US_1, RevAiDeployment.Customer`. I guess for the customer case, we probably wouldn't want to add a constant for either strategy, so they'd either have to modify internals or pass the prefix/url explicitly. In this case maybe we should do url explicitly, but I worry we may be putting the cart before the horse, as we still do not have any customer specific deployments.